### PR TITLE
Spark sql sentences added in the policy definition

### PR DESCRIPTION
**Pull request by compae**
## Description

Added functionality for execute sql sentences with Spark SQL when the policy starts. Now is possible make joins with triggers data and batch tables.

In policy definition:

`CREATE TEMPORARY TABLE mongoTable USING com.stratio.datasource.mongodb OPTIONS (host 'localhost:27017', database 'sparta', collection 'batchTable')`

Trigger definition:

`select * from mongoTable join stream on mongoTable.id = stream.id where mongoTable.response='yes' and stream.city='Madrid'`

### CODE REVIEW

File: serving-core/src/main/scala/com/stratio/sparta/serving/core/models/AggregationPoliciesModel.scala [hunk 1]
```diff
@@ -42,7 +42,8 @@ case class AggregationPoliciesModel(
                                      fragments: Seq[FragmentElementModel],
                                      userPluginsJars: Seq[String],
                                      remember: Option[String] = None,
-                                     sparkConf: Seq[SparkProperty] = Seq.empty[SparkProperty]
+                                     sparkConf: Seq[SparkProperty] = Seq.empty[SparkProperty],
```
Comment (danielcsant) [inline serving-core/src/main/scala/com/stratio/sparta/serving/core/models/AggregationPoliciesModel.scala]: I think these two new properties should be optional. Not being optional breaks regression with previous version

File: driver/src/main/scala/com/stratio/sparta/driver/cube/CubeWriter.scala [hunk 2]
```diff
@@ -48,7 +49,8 @@ case class CubeWriter(cube: Cube,
       toRow(dimensionValuesTime, measuresValues)
     }.foreachRDD(rdd => {
       if (rdd.take(1).length > 0) {
-        val sqlContext = SQLContext.getOrCreate(rdd.context)
+        //val sqlContext = SQLContext.getOrCreate(rdd.context)
```
Comment (danielcsant) [inline driver/src/main/scala/com/stratio/sparta/driver/cube/CubeWriter.scala]: check commented code in every file

**Comment by danielcsant** [comment]
I do not see this feature in documentation

**Comment by compae** [comment]
:shipit:

**Comment by stratiocommit** [comment]
A job will handle this PR ASAP on [Jenkins](http://jenkins.stratio.com/job/Sparta/job/CD/3231/)

**Comment by stratiocommit** [comment]
Rebase && testing in progress

**Pull request merged**